nom-derive-impl/Cargo.toml

```diff
@@ -3,33 +3,34 @@ name = "nom-derive-impl"
 description = "Custom derive nom parsers from struct"
 version = "0.10.0"
 license = "MIT/Apache-2.0"
-keywords = ["parser","nom"]
+keywords = ["parser", "nom"]
 categories = ["parsing"]
 authors = ["Pierre Chifflier <chifflier@wzdftpd.net>"]
 homepage = "https://github.com/rust-bakery/nom-derive"
 repository = "https://github.com/rust-bakery/nom-derive.git"
 readme = "../README.md"
-edition = "2018"
+edition = "2024"
 
 include = [
-  "LICENSE-*",
-  "README.md",
-  ".gitignore",
-  ".travis.yml",
-  "Cargo.toml",
-  "src/*.rs",
-  "src/gen/*.rs",
-  "src/meta/*.rs",
-  "tests/*.rs",
+    "LICENSE-*",
+    "README.md",
+    ".gitignore",
+    ".travis.yml",
+    "Cargo.toml",
+    "src/*.rs",
+    "src/gen/*.rs",
+    "src/meta/*.rs",
+    "tests/*.rs",
 ]
 
 [lib]
 proc-macro = true
 
 [dependencies]
-syn = { version="1.0.58", features=["parsing","extra-traits","full"] }
+syn = { version = "1.0.58", features = ["parsing", "extra-traits", "full"] }
```

Comment:
This may be done in another PR, but `syn` should be updated. I will take care of that